### DISCUSSION TITLE
Using DB4S for editing Kodi databases (was: fortschrittsanzeige)

### DISCUSSION THREAD

**Discussion by WernerSchmitt** [sqlitebrowser/sqlitebrowser#3431] [Q&A]

**Comment by justinclift** [comment]
Um, did you need help with something? :smile:

**Comment by WernerSchmitt** [comment]
Hello Justin,

thank you for your answer. I’m looking for a possibility to see what is the progress in my work. I’m writing in my database and when I look at the sheet, I never know, how long I have to wait till I can work with another Sheet in db browser..

I saw in a forum an answer, but I don’t know where or how I can fill in the script in Terminal, or another place.  I’m working in a kodi database mymusic82.db. 


Another question is, why I can’t change some entries in „Albumartistsort"? I’ve closed Kodi and I’m wondering why some Fields in my sheet are locked? 

Sorry for my bad english….

Greetings

Fredi

**Comment by justinclift** [comment]
Hmmm, we have several German speaking developers and Community members around.  So, I'll defer to them as this sounds like something application specific.

They might not get back to you by tomorrow though.

**Comment by FriedrichFroebel** [comment]
It sounds like this has something to do with the media center application Kodi. They apparently use an SQLite database and refer to DB4S for editing: https://kodi.wiki/view/Databases

Apart from that: I have never used Kodi and therefore I cannot say anything about application-specific stuff, although I at least understand German ;) I am going to write this in English, but feel free to answer in German it its easier for you to explain.

> I’m writing in my database and when I look at the sheet, I never know, how long I have to wait till I can work with another Sheet in db browser..

What exactly do you mean by "sheet"? A single database table? What exactly are you doing where it is relevant that you know the progress?

> I saw in a forum an answer, but I don’t know where or how I can fill in the script in Terminal, or another place.  I’m working in a kodi database mymusic82.db.

Without a link, it is hard to understand/grasp what you are referring to in this case. Nevertheless, Kodi maintainers itself seem to indicate that you should have enough understanding on relational databases and SQL before starting to "hack" anything on the database.

> Another question is, why I can’t change some entries in „Albumartistsort"? I’ve closed Kodi and I’m wondering why some Fields in my sheet are locked?

Which version of DB4S are you using on which operation system? Have you tried using the latest nightly versions? Usually, either the whole database file or nothing inside the database file should be locked. Partially locked fields sound rather strange. Do you have a screenshot on how such fields look? Or are you just getting an error when trying to change the field values?

**Comment by justinclift** [comment]
Oh, the media playing thing.  With that in mind, please make sure any screenshots don't contain stuff that would get us in trouble. :smile:
